tests/Backend/CommonPart1/TableWithConfigurationLoadTest.php

```diff
@@ -435,4 +446,121 @@ public function testLoadFromFileToTableOnError(): void
+        "description": ""
+      },
+      {
+        "sql": "CREATE TABLE {{ id(destSchemaName) }}.{{ id(destTableName ~ rand ~ '_tmp') }} ([id] NVARCHAR(4000), [NAME] NVARCHAR(4000)) WITH (DISTRIBUTION = ROUND_ROBIN,CLUSTERED COLUMNSTORE INDEX)",
+        "description": ""
+      },
+      {
+        "sql": "BEGIN TRANSACTION",
+        "description": ""
+      },
+      {
+        "sql": "UPDATE {{ id(destSchemaName) }}.{{ id(destTableName) }} SET [NAME] = COALESCE([src].[NAME], '') FROM {{ id(destSchemaName) }}.{{ id(stageTableName) }} AS [src] WHERE {{ id(destSchemaName) }}.{{ id(destTableName) }}.[id] = [src].[id] AND (COALESCE(CAST({{ id(destSchemaName) }}.{{ id(destTableName) }}.[NAME] AS NVARCHAR), '') != COALESCE([src].[NAME], '')) ",
+        "description": ""
+      },
+      {
+        "sql": "DELETE {{ id(destSchemaName) }}.{{ id(stageTableName) }} WHERE EXISTS (SELECT * FROM {{ id(destSchemaName) }}.{{ id(destTableName) }} WHERE {{ id(destSchemaName) }}.{{ id(destTableName) }}.[id] = {{ id(destSchemaName) }}.{{ id(stageTableName) }}.[id])",
+        "description": ""
+      },
+      {
+        "sql": "INSERT INTO {{ id(destSchemaName) }}.{{ id(destTableName ~ rand ~ '_tmp') }} ([id], [NAME]) SELECT a.[id],a.[NAME] FROM (SELECT [id], [NAME], ROW_NUMBER() OVER (PARTITION BY [id] ORDER BY [id]) AS \"_row_number_\" FROM {{ id(destSchemaName) }}.{{ id(stageTableName) }}) AS a WHERE a.\"_row_number_\" = 1",
+        "description": ""
+      },
+      {
+        "sql": "INSERT INTO {{ id(destSchemaName) }}.{{ id(destTableName) }} ([id], [NAME]) (SELECT CAST(COALESCE([id], '') as NVARCHAR) AS [id],CAST(COALESCE([NAME], '') as NVARCHAR) AS [NAME] FROM {{ id(destSchemaName) }}.{{ id(destTableName ~ rand ~ '_tmp') }} AS [src])",
+        "description": ""
+      },
+      {
+        "sql": "COMMIT",
+        "description": ""
+      }
+    ]
+  }
+}
+JSON;
+
+        [$tableId,] = $this->createTableWithConfiguration($json, $tableName, 'ingestionIncrementalLoad', [
+            [
+                'sql' => /** @lang TSQL */
+                    <<<SQL
+                    CREATE TABLE {{ id(bucketName) }}.{{ id(tableName) }} ([id] INTEGER, [NAME] VARCHAR(100))
+                    SQL,
+                'description' => 'first ever',
+            ],
+            [
+                'sql' => /** @lang TSQL */
+                    <<<SQL
+                    INSERT INTO {{ id(bucketName) }}.{{ id(tableName) }} ([id], [NAME]) SELECT 0, '- unchecked -';
+                    INSERT INTO {{ id(bucketName) }}.{{ id(tableName) }} ([id], [NAME]) SELECT 26, 'czech';
+                    INSERT INTO {{ id(bucketName) }}.{{ id(tableName) }} ([id], [NAME]) SELECT 1, 'english';
+                    INSERT INTO {{ id(bucketName) }}.{{ id(tableName) }} ([id], [NAME]) SELECT 11, 'finnish';
+                    INSERT INTO {{ id(bucketName) }}.{{ id(tableName) }} ([id], [NAME]) SELECT 24, 'french';
```

Comment:
Tohle dost možná funguje jen náhodou. Tipuju, že migrace počítají s tím, že jedna query = jedna položka v poli. Rozdělil bych to na jednotlivé query

Reply:
rozdelit inserty

Reply:
Udělal jsem